src/OrchardCore.Modules/OrchardCore.Layers/Controllers/AdminController.cs

```diff
@@ -104,18 +104,19 @@ public async Task<IActionResult> Index(LayersIndexViewModel model)
             return RedirectToAction("Index");
         }
 
-        public async Task<IActionResult> Create()
+        public async Task<IActionResult> Create(string returnUrl = null)
         {
             if (!await _authorizationService.AuthorizeAsync(User, Permissions.ManageLayers))
             {
                 return Forbid();
             }
 
+            ViewData["ReturnUrl"] = returnUrl;
```

Comment:
Is it necessary?

Reply:
No, it wasn't necessary. Changed to not use ViewData.